assets/javascripts/discourse/initializers/extend-for-solved-button.js.es6

```diff
@@ -185,6 +184,19 @@ function initializeWithApi(api) {
     );
   });
 
+  api.addAdvancedSearchOptions({
+    statusOptions: [
+      {
+        name: I18n.t("search.advanced.statuses.solved"),
+        value: "solved",
+      },
+      {
+        name: I18n.t("search.advanced.statuses.unsolved"),
+        value: "unsolved",
+      },
+    ],
+  });
+
```

Comment:
Can you move this into its own `initialize` function with the new version number of the plugin API? Otherwise it will break on the stable version of Discourse